tools/codemod/src/v4.0.0/__testfixtures__/frontstage-to-config/Frontstage1.output.tsx

```diff
@@ -4,8 +4,8 @@
 *--------------------------------------------------------------------------------------------*/
```

Comment:
How is this different from `Frontstage.output.tsx`? File names should be descriptive as well.

Reply:
It is based on filenames that already exist in the codebase (i.e. StandardFrontstageProvider, Frontstage1, Frontstage2 ...)